release-notes/v2.1.0.md

```diff
@@ -0,0 +1,36 @@
+# HPE CSI Driver for Kubernetes 2.1.0 Release Notes
+
+## HPE CSI Driver for Kubernetes 2.1.0
+
+| **Version:** |2.1.0|
+|--------------|-----|
+| **Revision:** | Tuesday January 10, 2022 |
+
+## Resolved Issues
+
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v2.1.0.
+
+|ID|Component |Title|Resolution|
+|--|---------|-----|-----------|
+
+The following table lists the resolved issues for HPE Alletra 9000 and Primera/3PAR CSP for HPE CSI Driver release 2.1.0.
+
+|ID|Component |Title|Resolution|
+|--|---------|-----|-----------|
+
+
+## Known Issues
+
+The following table lists the known issues for HPE CSI Driver for Kubernetes v2.1.0. Please note that, `Known Issues` from previous releases are still applicable with suggested workarounds, if they are not part of `Resolved Issues` above.
+
+|ID|Component |Title|Description|Workaround|
+|--|---------|-----|-----------|----------|
+
+
+The following table lists the known issues for HPE Alletra 9000 and Primera/3PAR CSP for HPE CSI Driver release 2.1.0
+
+|ID|Component |Title|Description|Workaround|
+|--|---------|-----|-----------|----------|
+|CON-2330|csi.k8s|Discrepancy in volume size between array and master. | When volume size is not provided in multiple of 256 then array round it off to nearby 256 multiple value but on the node level it's size get not reflected. |None|
+|CON-2536|csi.k8s| RCG Greater than 23 characters generating errors. | When user provides RCG name more than 23 characters, array does not create rcg as it is against it's core behaviour |RCG name should always be less than 23 characcters. |
+|CON-2463|csi.k8s|mutate pvc with oneRgcPerPvc: "!23false##", no further edits working and csp logs are looping continuously. | Not able to edit once incorrect values are provided. |As a solution we just needs to provide empty string for all the annotation related to remotecopy group mutation such as remoteCopyGroup, oneRcgPerPvc, replicationDevices then again user needs to enter the correct values . so this is a workaround for now. |
```

Comment:
Title : Mutating the oneRcgPerPvc attribute with invalid value (such as "!23false##") would render any subsequent mutations ineffective
Description: In addition the logs show continous activity. No feedback provided to kubernetes admin/user.
Workaround: Mutate the PVC with the annotation values of the attributes - remoteCopyGroup, oneRcgPerPvc, replicationDevices set to empty, and then mutate again to provide the accurate values for them.

Reply:
modified the same content